Src/PCompiler/PCommandLine/Options/PCheckerOptions.cs

```diff
@@ -136,6 +136,10 @@ private static void UpdateConfigurationWithParsedArgument(CheckerConfiguration c
                     break;
                 case "timeout":
                     checkerConfiguration.Timeout = (int)(uint)option.Value;
+                    if (checkerConfiguration.TestingIterations == 1)
```

Comment:
What is this change doing? please explain.

Reply:
If only a timeout is passed, P CLI disables iteration limit (sets to 0).
If a timeout is passed followed by non-default iteration limit, both timeout and iteration limit are enforced.

If only iteration limit is passed, iteration limit is enforced with no timeout.